gffquant/profilers/feature_quantifier.py

```
@@ -330,6 +312,44 @@ def count_alignments(
+            # )
+
+        for metric, value in (
+                ("Input reads", "full_read_count"),
+                ("Aligned reads", "read_count"),
+                ("Alignments", "pysam_total"),
+                ("Reads passing filters", "filtered_read_count"),
+                ("Alignments passing filters", "pysam_passed"),
+                ("  - Discarded due to seqid", "pysam_seqid_filt"),
+                ("  - Discarded due to length", "pysam_len_filt"),
+                # ("Unannotated multimappers", "unannotated_ambig"),
+            ):
```

Flake8 (7.3.0)

[error] continuation line under-indented for hanging indent

(E121)